File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/per.php
<?php

fscanf(STDIN, "%d%d", $l, $r);
$val = 2 * max($l, $r);
if ($val == 0)
   echo "Not a moose";
elseif ($l == $r)
   echo "Even $val";
else
   echo "Odd $val";

File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/per.scala
object per {
    def main(args: Array[String]) {
        var line = scala.io.StdIn.readLine()
        val Array(l, r) = line.split(" ").map(_.toLong)
        if (l == 0 && r == 0) {
            Console.out.println("Not a moose");
        } else if (l == r) {
            Console.out.println("Even " + (l+r));
        } else {
            Console.out.println("Odd " + 2*Math.max(l,r));
        }
   }
}
